### DISCUSSION TITLE
Icon Request: fa-desk

### DISCUSSION THREAD

**Discussion by meyersbs** [FortAwesome/Font-Awesome#21114] [Icon Requests]
Something simple like this: 
![image](https://cloud.githubusercontent.com/assets/4073851/16456481/fc8dd27e-3de5-11e6-94bc-ca6683a07070.png)

**Comment by tagliala** [comment]
I'm going to consider this a duplicate of #3984. There is an example in #3986 which is better than this one because it suits 14px (only a desk and a monitor)

Please +1 that request and post there your example

**Comment by tagliala** [comment]
Reconsidering. Leave this opened as office-desk. the other one opened as office-chair

**Comment by casman300** [comment]
+1

**Comment by vheimpel** [comment]
+1

**Comment by dewabharata** [comment]
+1

**Comment by savadodesigns** [comment]
+1

**Comment by krystian-dev548** [comment]
+1! This should be a must

**Comment by krystian-dev548** [comment]
Any update about this great icon? :)

**Comment by Jatin-8898** [comment]
+1

**Comment by spmsh** [comment]
+1

or simpler, the keynote icon without the microphone is looking like a simple desk/table. thanks.

**Comment by DepaMarco** [comment]
+1, thanks

**Comment by spmsh** [comment]
+1

**Comment by roshanzaid** [comment]
+1

**Comment by savadodesigns** [comment]
Guys? It's been 4 years now...

**Comment by stphnwlkr** [comment]
+1

**Comment by spmsh** [comment]
+1

**Comment by savadodesigns** [comment]
+1 AGAIN

**Comment by savadodesigns** [comment]
It's been almost 6 years... In the meantime the most useless icons have been added, but we still cannot show a desk... What the bleep...

**Comment by spmsh** [comment]
the usual bump. this is much needed in the post pandemic word where desk hoteling becomes more popular in the office world.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
Added to 7.2: https://fontawesome.com/icons/desk?f=classic&s=solid
